.github/CODEOWNERS

```diff
@@ -1,6 +1,9 @@
 # NHS Notify Code Owners
 
-* @rossbugginsnhs @m-houston @edmundcraske2-nhs @timireland
+* @rossbugginsnhs @m-houston @aidenvaines-bjss @timireland
+
+# Notify platform owners
+/infrastructure/terraform/  @NHSDigital/nhs-notify-platform
```

Comment:
This team doesn't exist yet - do you want to create it, or does it need an admin to create a sub-team? I think we've previously used the `nhs-notify-amet` group for admin reviews, but possibly should retire or rename that group now.